### PR TITLE
SERVER-84675 Add workloads running CollScan queries on various types

### PR DESCRIPTION
**Pull request by RuoxinXu**
Thanks for submitting a PR to the Genny repo. Please include the following fields (if relevant) prior to submitting your PR.

**Jira Ticket:** SERVER-84675

**Whats Changed:**  
A new genny workload that runs CollScan queries

**Patch Results:**
https://spruce.mongodb.com/version/65af7fd7c9ec441cbae823b3/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC

### CODE REVIEW

File: src/phases/query/CollScanOnMixedDataTypes.yml [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+    OnlyActiveInPhases:
+      Active: [3]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Collection: *Collection
+        Operations:
+        - OperationName: find
+          OperationCommand:
+            Filter: { "numInt": {$gt: 10} }
```
Comment (alyacb) [inline src/phases/query/CollScanOnMixedDataTypes.yml:126]: Are all these queries meant to have approximately the same cardinality? If not, how were the constants selected for each individual query?

**Review by alyacb — APPROVED** [review]
LGTM, mod a Q about how we picked the queries

File: src/phases/query/CollScanOnMixedDataTypes.yml (hunk 1 above)
Comment (RuoxinXu) [reply src/phases/query/CollScanOnMixedDataTypes.yml:126]: Good point. Basically I just chose the value in the middle of the corresponding range. It should have "approximately" the same cardinality, but I'm gonna use the min/max value of the range to maximize the cardinality so it should be stable at least.

**Comment by RuoxinXu** [comment]
evergreen merge

**Pull request merged**